src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -110,6 +111,35 @@ const SubmissionDetails = _.flow(
   )(workflows)
 
   const { succeeded, failed, running, submitted } = _.groupBy(wf => collapseStatus(wf.status), workflows)
+
+  const archiveLimitYears = 6
```

Comment:
The AC for this ticket had this bullet:
"comment everywhere we have the deletion delay constant in code, a pointer to all the other locations this constant exists in code"
I'd love to see a comment here that points to the other repositories that this constant exists as well as a comment in that code that points here.